README.md

```diff
@@ -17,16 +17,24 @@ Hyprland's simple, yet multi-threaded and GPU-accelerated screen locking utility
 ## Building
 
 ### Deps
+Hyprlock requires `hyprlang` >= 0.4.0. See the 
+[README](https://github.com/hyprwm/hyprlang/blob/main/README.md#building-and-installation) for 
+build instructions.
```

Comment:
why this part? Most package managers are now on >= 0.4.0

Reply:
Yeah I think the build from source part wasn't actually required as it is already shipped by default in most distros repos. Maybe we can rephrase it to say that if its not shipped on any distro's repos, people can still jump to its build instructions. I will put it in my next commit.

Reply:
that's default for any dep, I think the comment is redundant.

Reply:
Ok. I changed it in my latest commit.